### ISSUE TITLE
Accessibility Insights Fastpass causes pages using the react monaco editor to crash

### ISSUE THREAD

**Issue by mryanlo** [react-monaco-editor/react-monaco-editor#354]
Repro steps:
1. Open a page in a react app that uses the react monaco editor. For example: https://monaco-react.surenatoyan.com/
2. Install the [Accessibility Insights chrome extension](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni?hl=en-US)
3. Run the Fast Pass check on the react app page.
![image](https://user-images.githubusercontent.com/55516393/123315005-154c0b00-d4e0-11eb-9dd2-f7ee3a893ad8.png)
4. Observe the fast pass takes longer to run and after a few seconds, the page crashes with an out of memory error:
![image](https://user-images.githubusercontent.com/55516393/123315101-2e54bc00-d4e0-11eb-979e-7bd69bbc5dde.png)

Expected: fast pass should be able to run on all parts of the page to test for accessibility issues even if the page has a monaco editor in it.
Actual: the out of memory error

We need to run this fast pass command on our app's pages to make sure they are accessibility compliant for our customers. We can't do so if the monaco editor is causing some to crash, and we're planning on adding the monaco editor to more parts of our app soon.

**Comment by mryanlo** [comment]
Update: this issue repros in Google Chrome. On Microsoft Edge, it takes a while to run, but it does succeed

**Comment by domoritz** [comment]
Could this be an issue with Monaco or React? Could you look into what's causing the issue?

**Comment by mryanlo** [comment]
@domoritz thank you for your response. Yes, I meant to update this with some more findings i discovered yesterday. A teammate and I synced and we discovered this issue repros with the microsoft monaco editor package that the react monaco editor uses as a whole. https://github.com/microsoft/monaco-editor/issues/2546

In the linked issue, if you open the test microsoft monaco editor playground, then you'll find that this issue repros there. So it's definitely an issue with monaco and not react.

**Comment by domoritz** [comment]
Thanks for the investigation. I'll close the issue here then.
